features/FxA-45-device-registration-api/README.md

```diff
@@ -81,11 +71,6 @@ in the future.
   and be signed out of all devices,
   including the current device.
 
-* As a signed-in user,
-  I want to update my device details
-  and see the newly-changed details
-  automatically reflected in all open instances of the devices view.
-
```

Comment:
Or, is is de-scoping an implied use of push and just using the webchannel stuff to update local devices view?